test/module/vendor/rocksdb_test.cpp

```diff
@@ -0,0 +1,269 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <rocksdb/db.h>
+
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+
+#include "ametsuchi/impl/rocksdb_common.hpp"
```

Comment:
Are you sure to add rocks_db under ametsuhi?

Reply:
this is the gate for db and storage

Reply:
This is not a "vendor" test if you are using ametsuchi. This test should be either placed in ametsuchi directory, since it uses our code, or there should be a dedicated test which only shows sample usage of RocksDB API, without our ametsuchi code.